### ISSUE TITLE
error importing vue2-leaflet module

### ISSUE THREAD

**Issue by ellisdod** [tesselo/vue2-leaflet-vectorgrid#6]
I can't get this to work:
`import v2l from 'vue2-leaflet'`

v2l is undefined and it throws an exception.

**Comment by yellowcap** [comment]
Could you try `import Vue2Leaflet from 'vue2-leaflet'`?

**Comment by dGilli** [comment]
I get the same error even with the default import.

**Comment by SaboteurInCave** [comment]
Sorry for inconvenience, but it seems, that latest commit with dependecies fix is not uploaded on npm - and developers still load outdated package version. @yellowcap, can you, please, upload new version?

**Comment by vvavepacket** [comment]
@SaboteurInCave  I encounter the same error. It seems the developer hasnt updated npm. How can I fix it myself temporarily and just apply the necessary changes in my local machine?

 WARNING  Compiled with 1 warnings                                                                               9:38:52 AM

```
 warning  in ./node_modules/vue2-leaflet-vectorgrid/Vue2LeafletVectorGridProtobuf.vue?vue&type=script&lang=js&

"export 'default' (imported as 'v2l') was not found in 'vue2-leaflet'
```

**Comment by yellowcap** [comment]
I just pushed an update to npm now. Version 1.0.7. Does this solve your problems?

**Comment by vvavepacket** [comment]
@yellowcap 
I tried installing 1.0.7 from npm but got the following issue

```
 ERROR  Failed to compile with 1 errors                                                                         10:56:40 AM

This dependency was not found:

* leaflet.vectorgrid in ./node_modules/cache-loader/dist/cjs.js??ref--12-0!./node_modules/babel-loader/lib!./node_modules/c
ache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./node_modules/vue2-leaflet-vectorgrid/
Vue2LeafletVectorGridProtobuf.vue?vue&type=script&lang=js&

To install it, you can run: npm install --save leaflet.vectorgrid
```

I mean I should not install leaflet.vectorgrid in my main project since this library should take care, but it seems I'm getting the error now when I installed 1.0.7

**Comment by yellowcap** [comment]
I made a few more improvements and released them under 1.1.1. Please try again and let me know if you still get that error.

**Comment by vvavepacket** [comment]
Yes 1.1.1 fixed the issue. Thanks!
